index.html

```diff
@@ -0,0 +1,208 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="style.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homework</title>
+  </head>
+  <body>
+    <main>
+      <header>
+        <section>
+          <nav>
+            <div>
+              <img src="icons/Logo.svg" alt="logo" />
+            </div>
+          </nav>
+        </section>
+        <section>
+          <ul>
+            <li>Product</li>
+            <li>Customers</li>
+            <li>Pricing</li>
+            <li>Resources</li>
+          </ul>
+        </section>
+        <section>
+          <button type="button" class="btn btn-secondary">Sign in</button>
+          <div>
+            <button type="button" class="btn btn-warning">Sign up</button>
+          </div>
+        </section>
+        <section>
+          <h1>
+            The Design Thinking <br />
+            superpowers
+          </h1>
+          <p>
+            Tools, tutorials, design and innovation experts, all <br />
+            in one place! The most intuitive way to imagine <br />
+            your next user experience.
+          </p>
+          <div class="container">
+            <a href="#" type="link">
+              <img
+                src="icons/button.svg"
+                alt="Rectangle"
+                width="175px"
+                height="53px"
+              />
+            </a>
+          </div>
+        </section>
+        <section>
+          <a href="#" type="link">
+            <img src="icons/buttonvideo.svg" alt="video" />
+          </a>
+        </section>
+        <section>
+          <aside>
+            <object type="image/svg+xml" data="images/image.svg" class="image">
+              image
+            </object>
+          </aside>
+        </section>
+      </header>
+      <section>
+        <h1>We design tools to unveil your superpowers</h1>
+        <img src="icons/icon1.svg" alt="click" />
+        <h2>First click test</h2>
+        <p>While most people enjoy casino gambling.</p>
+      </section>
+      <section>
+        <img src="icons/icon2.svg" alt="icon2" />
+        <h2>Design surveys</h2>
+        <p>Sports betting, lottery and bingo playing for the fun</p>
+      </section>
+      <section>
+        <img src="icons/icon3.svg" alt="icon3" />
+        <h2>Preference tests</h2>
+        <p>The Myspace page defines the individual.</p>
+      </section>
+      <section>
+        <img src="icons/icon4.svg" alt="icon4" />
+        <h2>Five second tests</h2>
+        <p>Personal choices and the overall personality of the person.</p>
+      </section>
```

Comment:
Якщо елементи мають дуже схожий вигляд, то це скоріш за все має бути список

```suggestion
        <h1>We design tools to unveil your superpowers</h1>
        <ul>
          <li>
            <img src="icons/icon1.svg" alt="click" />
            <h2>First click test</h2>
            <p>While most people enjoy casino gambling.</p>
          </li>
          <li>
            <img src="icons/icon2.svg" alt="icon2" />
            <h2>Design surveys</h2>
            <p>Sports betting, lottery and bingo playing for the fun</p>
          </li>
          <li>
            <img src="icons/icon3.svg" alt="icon3" />
            <h2>Preference tests</h2>
            <p>The Myspace page defines the individual.</p>
          </li>
          <li>
            <img src="icons/icon4.svg" alt="icon4" />
            <h2>Five second tests</h2>
            <p>Personal choices and the overall personality of the person.</p>
          </li>
        </ul>
```